Jenkinsfile

```diff
@@ -0,0 +1,18 @@
+#!groovy
+node('xenial') {
+  try {
+	stage('Checkout') {
+	  clean_checkout()
+	}
+    stage('Build') {
+        sh 'git clone git@github.com:PolySync/oscc.git --branch master && mkdir build && cd build && cmake .. -DKIA_SOUL=ON && make'
+        echo 'Build Complete!'
+    }
+  }
+  catch(Exception e) {
```

Comment:
Does the code in the `try` block potentially throw a checked exception? If not, can we elide this `catch` block?

Reply:
Any of the commands in the shell script could *potentially* fail, so I want to make sure the directory isn't deleted. I'm not sure if the `catch` block is specifically needed, but I think the finally block should certainly be there to ensure we cleanup.

Reply:
I was a skeptic about dropping the catch but found it to work from a similar comment when ROSCCO was being reviewed. https://github.com/PolySync/roscco/blob/devel/Jenkinsfile Apparently try/finally is a thing...

Reply:
Alrighty! I'll try it without.

Reply:
Removed as of 2f1377c